### DISCUSSION TITLE
gifting issue

### DISCUSSION THREAD

**Discussion by TheAwesomePika** [Monika-After-Story/MonikaModDev#9715] [QA - tech support + general]
whenever i try to gift monika the 2B halloween outfit it says she cant read the file can someone help me with this please
![Screenshot (29072)](https://user-images.githubusercontent.com/117291306/199517321-0dba37d1-cfd4-495d-95cd-d9e75d35aeff.png)
![Screenshot (29074)](https://user-images.githubusercontent.com/117291306/199517681-2e1f556a-d7aa-48db-8782-8dc2945c38ac.png)

**Comment by Booplicate** [comment]
Follow [this](https://www.youtube.com/watch?v=bjJ0U8l1m8c) tutorial.
